broker/src/main/java/io/camunda/zeebe/broker/system/configuration/FeatureFlagsCfg.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.broker.system.configuration;
+
+import io.camunda.zeebe.util.FeatureFlags;
+import org.apache.commons.lang3.builder.ToStringBuilder;
+import org.apache.commons.lang3.builder.ToStringStyle;
+
+public class FeatureFlagsCfg {
```

Comment:
```suggestion
public final class FeatureFlagsCfg {
```

Reply:
Seems like having no properties causes Jackson to fail in serializing this. I think this can be circumvented by configuring the mapper to disable `SerializationFeature.FAIL_ON_EMPTY_BEANS`. I don't foresee any big downsides, but maybe I'm missing something.

Reply:
Do you know a way to do this on a class level? I googled a bunch, but are none the wiser.

Reply:
Nevermind, found a way.

Reply:
Apparently just adding `@JsonSerialize` on top makes it work :+1: Perhaps annotating it here makes it use a different default serializer? It's unclear to me. But adding it, it then behaves as expected with or without fields. Maybe we just add a comment explaining this is needed for when the configuration class is empty.

Reply:
Sorry, hadn't seen your comment :sweat_smile: